### PR TITLE
jamovi: Add version 1.2.27.0

### PR DESCRIPTION
**Pull request by adrianocorrea**

### CODE REVIEW

File: bucket/jamovi.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.6.8.0",
+    "description": "Free and open statistical software to bridge the gap between researcher and statistician",
+    "homepage": "https://www.jamovi.org",
+    "license": {
+        "identifier": "AGPL3",
```
Comment (Ash258) [inline bucket/jamovi.json:6]: Invalid SPDX identifier

File: bucket/jamovi.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "url": "https://github.com/jamovi/jamovi/blob/master/LICENSE.md"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://www.jamovi.org/downloads/jamovi-1.6.8.0-win64.zip",
+            "hash": "09CED789F6FD336C37FC3A6DA4A93BA9CA7E3BB53C54B5692EE9AA0E0F1FECFB"
+        }
+    },
+    "extract_dir": "jamovi-1.6.8.0-win64",
```
Comment (Ash258) [inline bucket/jamovi.json:15]: This is clearly 64bit only

File: bucket/jamovi.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "description": "Free and open statistical software to bridge the gap between researcher and statistician",
+    "homepage": "https://www.jamovi.org",
+    "license": {
+        "identifier": "AGPL3",
+        "url": "https://github.com/jamovi/jamovi/blob/master/LICENSE.md"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://www.jamovi.org/downloads/jamovi-1.6.8.0-win64.zip",
+            "hash": "09CED789F6FD336C37FC3A6DA4A93BA9CA7E3BB53C54B5692EE9AA0E0F1FECFB"
```
Comment (Ash258) [inline bucket/jamovi.json:12]: lowercase

File: bucket/jamovi.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    "bin": "bin\\jamovi.exe",
+    "shortcuts": [
+        [
+            "bin\\jamovi.exe",
+            "Jamovi"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.jamovi.org/download.html",
+        "regex": "/downloads/jamovi-([\\d.]+)\\-win64.zip"
```
Comment (Ash258) [inline bucket/jamovi.json:25]: `\\.zip`

**Review by Ash258 — CHANGES_REQUESTED** [review]
Why do you prefer version, which is not recommended by authors??

image

File: bucket/jamovi.json (hunk 1 above)
Comment (adrianocorrea) [reply bucket/jamovi.json]: Updated identifier. See commit "80da546".

File: bucket/jamovi.json (hunk 2 above)
Comment (adrianocorrea) [reply bucket/jamovi.json]: Moved key to "architecture":"64bit".

File: bucket/jamovi.json (hunk 3 above)
Comment (adrianocorrea) [reply bucket/jamovi.json]: Updated for style consistency. Also updated for stable version (i.e. 1.2.27.0).

File: bucket/jamovi.json (hunk 4 above)
Comment (adrianocorrea) [reply bucket/jamovi.json]: Fixed regex.

File: bucket/jamovi.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.2.27.0",
+    "description": "Free and open statistical software to bridge the gap between researcher and statistician",
```
Comment (Ash258) [inline bucket/jamovi.json:3]:
```suggestion
    "description": "Statistical spreadsheet",
```

File: bucket/jamovi.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    "homepage": "https://www.jamovi.org",
+    "license": {
+        "identifier": "AGPL-3.0-only",
+        "url": "https://github.com/jamovi/jamovi/blob/master/LICENSE.md"
+    },
```
Comment (Ash258) [inline bucket/jamovi.json:8]:
```suggestion
    "license": "AGPL-3.0-only",
```

File: bucket/jamovi.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://www.jamovi.org/downloads/jamovi-1.2.27.0-win64.zip",
+            "hash": "7c15ce84a2a1b7a1ba166b61ea2e7f2559215dcc5c92a571d865502d3cbefceb",
+            "extract_dir": "jamovi-1.2.27.0-win64"
+        }
+    },
+    "bin": "bin\\jamovi.exe",
```
Comment (Ash258) [inline bucket/jamovi.json:16]:
```suggestion
```

File: bucket/jamovi.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    "shortcuts": [
+        [
+            "bin\\jamovi.exe",
+            "Jamovi"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.jamovi.org/download.html",
+        "regex": "/downloads/jamovi-([\\d.]+)\\-win64\\.zip"
```
Comment (Ash258) [inline bucket/jamovi.json:25]:
```suggestion
        "regex": "(?sm)/downloads/jamovi-([\\d.]+)\\-win64\\.exe.*?solid"
```

File: bucket/jamovi.json [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+              "url": "https://www.jamovi.org/downloads/jamovi-$version-win64.zip"
```
Comment (Ash258) [inline bucket/jamovi.json:30]:
```suggestion
                "url": "https://www.jamovi.org/downloads/jamovi-$version-win64.zip"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**